test/TestUtils.py

```diff
@@ -154,3 +155,15 @@ def test_normpath_with_string(self):
         self.assertEqual(
             utils.normpath("a/b/../"),
             "a")
+
+    def test_expand_path_vars(self):
+        test_path = '/test/path'
+        os.environ['TEST_PATH'] = test_path
```

Comment:
The method you used modifies the global env vars object for the whole process, affecting other tests.

Please use a built-in monkeypatch fixture to set the env var temporary just for this test. It'll automatically unset it on exit.
```suggestion
    monkeypatch.setenv('TEST_PATH', test_path)
```